prac_06/guitars.py

```diff
@@ -2,26 +2,25 @@
 Time expected: 30 min
        actual:  min
 """
+from prac_06.guitar import Guitar
 
 
 def main():
     """Ask user for guitar data and then present information
     in a nicely formatted table."""
-    print("My guitars!")
+    guitars = []
 
-    name = get_non_blank_string("Name: ")
-    year = get_positive_integer("Year: ")
-    cost = get_positive_float("Cost: $")
+    print("My guitars!")
+    name = input("Name: ").strip()
+    while name != "":
+        year = get_positive_integer("Year: ")
+        cost = get_positive_float("Cost: $")
 
+        guitar = Guitar(name, year, cost)
+        guitars.append(guitar)
+        print(f"{guitar} added.")
```

Comment:
Does this match the output they asked for exactly?

Reply:
Hi @Brennen12 . It actually does! `print(f"{guitar}")` prints the guitar object using the `__str__` method.